js/constants/messages.js

```diff
@@ -110,6 +110,9 @@ const messages = {
   RESPONSE_WINDOW_STATE: _,
   LAST_WINDOW_STATE: _,
   UNDO_CLOSED_WINDOW: _,
+  // Menu rebuilding
+  REQUEST_WINDOW_STATE_FOR_MENU: _,
+  RESPONSE_WINDOW_STATE_FOR_MENU: _,
```

Comment:
If you're only using closedFrames from windowState then I think you should call this:
REQUEST_WINDOW_CLOSED_FRAMES
RESPONSE_WINDOW_CLOSED_FRAMES

And then update accordingly.